triggers-deploy.patch

```diff
@@ -1,3 +1,16 @@
+diff --git a/pkg/reconciler/eventlistener/resources/config.go b/pkg/reconciler/eventlistener/resources/config.go
+index 20b4bfc6..c74da9f7 100644
+--- a/pkg/reconciler/eventlistener/resources/config.go
++++ b/pkg/reconciler/eventlistener/resources/config.go
+@@ -22,7 +22,7 @@ var (
+ 	// DefaultPort is the port used by default.
+ 	DefaultPort = 8080
+ 	// DefaultSetSecurityContext is the SetSecurityContext value used by default.
+-	DefaultSetSecurityContext = true
++	DefaultSetSecurityContext = false
```

Comment:
Instead of this, we should try to do changes in `config/controller.yaml`.
```
          "-el-security-context=false",

Reply:
Updated it.
Yes, we are already using this flag. It was redundant to do it here. I guess I forgot to remove this.